cluster/examples/kubernetes/ceph/create-external-cluster-resources.py

```diff
@@ -403,15 +403,18 @@ def get_active_and_standby_mgrs(self):
             monitoring_endpoint_port = self.DEFAULT_MONITORING_ENDPOINT_PORT
 
```

Comment:
If we can add the following lines before `try` (that is before line#405), we could completely avoid any other further changes (from line#406 on wards). 

```python
        # user could give comma and space separated inputs (like --monitoring-endpoint="<ip1>, <ip2>")
        monitoring_endpoint_ip_list = monitoring_endpoint_ip_list.replace(",", " ")
        monitoring_endpoint_ip_list_split = monitoring_endpoint_ip_list.split()
        # if monitoring-endpoint could not be found, raise an error
        if len(monitoring_endpoint_ip_list_split) == 0:
            raise ExecutionFailureException("No 'monitoring-endpoint' found")
        # first ip is treated as the main monitoring-endpoint
        monitoring_endpoint_ip = monitoring_endpoint_ip_list_split[0]
        # rest of ip-s are add to the 'standby_mgrs'
        standby_mgrs.extend(monitoring_endpoint_ip_list_split[1:])
```

Reply:
That works too! Thanks!